DuckDuckGo/PrivacyProtectionOverviewController.swift

```diff
@@ -173,13 +175,36 @@ class TrackerNetworkLeaderboardCell: UITableViewCell {
     @IBOutlet weak var firstPill: TrackerNetworkPillView!
     @IBOutlet weak var secondPill: TrackerNetworkPillView!
     @IBOutlet weak var thirdPill: TrackerNetworkPillView!
+    @IBOutlet weak var message: UILabel!
+    @IBOutlet weak var forwardArrow: UIImageView!
+
+    var leaderboard = NetworkLeaderboard.shared
 
     func didLoad() {
         firstPill.didLoad()
         secondPill.didLoad()
         thirdPill.didLoad()
     }
 
+    func update() {
+        let networksDetected = leaderboard.networksDetected()
+
+        firstPill.isHidden = networksDetected.count < 3
```

Comment:
would probably add a local boolean variable with the result of `networksDetected.count < 3` since it is used so much